modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1321,6 +1469,9 @@ bool CvCapture_FFMPEG::retrieveFrame(int, unsigned char** data, int* step, int*
     *height = frame.height;
     *cn = frame.cn;
 
+    if (sw_picture != picture) {
+        av_frame_unref(sw_picture);
```

Comment:
We have a build problem with older FFmpeg:
```
 In file included from ../opencv/modules/videoio/src/cap_ffmpeg.cpp:50:0:
 ../opencv/modules/videoio/src/cap_ffmpeg_impl.hpp: In member function 'bool CvCapture_FFMPEG::retrieveFrame(int, unsigned char**, int*, int*, int*, int*)':
 ../opencv/modules/videoio/src/cap_ffmpeg_impl.hpp:1473:34: error: 'av_frame_unref' was not declared in this scope
          av_frame_unref(sw_picture);
                                   ^
```
```
      avcodec:                   YES (54.35.0)
      avformat:                  YES (54.20.4)
      avutil:                    YES (52.3.0)
      swscale:                   YES (2.1.1)
      avresample:                YES (1.0.1)
```
Perhaps `libavutil/frame.h` should be included directly in this file.
**cc** @alalek